### ISSUE TITLE
please add in"Icon of CMD file"

### ISSUE THREAD

**Issue by 26chk** [Peltoche/lsd#483]
![Screenshot_20210209_174723](https://user-images.githubusercontent.com/61009000/107346305-807fdc80-6aff-11eb-933d-7dbf54ccb2b7.jpg)
![Screenshot_20210209_174738](https://user-images.githubusercontent.com/61009000/107346312-82e23680-6aff-11eb-977b-d2f2abdc73d8.jpg)

**Comment by meain** [comment]
Feel free to submit a PR to add icons ([ref](https://github.com/Peltoche/lsd/blob/841ad99ced2f3a0103869fee696693442a2f2b6a/src/icon.rs#L141)).

As for the color of `README.md` and `Makefile`, you can completely customize them using lscolors. eg: https://github.com/trapd00r/LS_COLORS

**Comment by 26chk** [comment]
> 随时提交PR以添加图标（[ref](https://github.com/Peltoche/lsd/blob/841ad99ced2f3a0103869fee696693442a2f2b6a/src/icon.rs#L141)）。
> 
> 至于颜色`README.md`和`Makefile`，则可以使用LSCOLORS完全自定义。例如：[https](https://github.com/trapd00r/LS_COLORS)：[//github.com/trapd00r/LS_COLORS](https://github.com/trapd00r/LS_COLORS)

Yes, that sounds great, can you underline some files？such as "README"and "MakeFile"?👀

**Comment by meain** [comment]
Yup, we should be able to do whatever lscolors can handle.
You can refer to documentation on lscolors [here](www.bigsoft.co.uk/blog/2008/04/11/configuring-ls_colors) and [here](www.bigsoft.co.uk/blog/2008/04/11/configuring-ls_colors). These should give you an idea on how to configure lscolors to your exact specific usecase.
